chats/apps/api/v1/tests/test_actions.py

```diff
@@ -0,0 +1,64 @@
+from django.urls import reverse
+
+from rest_framework.test import APITestCase
+from rest_framework.authtoken.models import Token
+
+from chats.apps.accounts.models import User
+from chats.apps.projects.models import Project
+
+
+
```

Comment:
3 blank lines, leave only two, following the pep8 pattern.
More details in: https://peps.python.org/pep-0008/